docs/create-handlers.md

```diff
@@ -3,14 +3,326 @@ id: create-handlers
+
+***For saves and testing:***
+Line | Description
+---- | -----------
+Game.KeepSymLinkOnExit = true; | False = after Nucleus is closed, game files inside Nucleus\content will be deleted. Enable = Nucleus will keep the files inside Nucleus\content even after is closed. Useful if the game store saves it in the game-folder.
+
+## Nucleus-Coop Environment
+
+This function will provide a different game's Document path for each player.
+
+When a player is initialized, Nucleus-Coop will edit the Windows registry lines to make the game use a different path for search/create Documents files.
+You can find the registry lines inside this path:
+**HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders**
+
+Then, you will be able to access in the script context part by using:
+- Context.EnvironmentPlayer (specific player Environment path) or  Context.EnvironmentRoot (Nucleus-Coop Environment Folder root) when using the UserProfile lines.
+- Context.DocumentsPlayer (specific player Documents path) or Context.DocumentsRoot (Nucleus-Coop Documents Folder root) when using the Documents lines.
+
+Line | Description
+---- | -----------
+Game.UseNucleusEnvironment = true; | Use custom environment variables for games that use them, replaces some common paths (e.g. AppData) with C:\Users\<your username>\NucleusCoop
+Game.UserProfileConfigPath = "AppData\\Local\\Game\\Config"; | Relative path from user profile (e.g. C:\Users\ZeroFox) to game's config path | Used to provide some extra functionality (open/delete/copy over to Nucleus Environment). In context = Context.UserProfileConfigPath 
+Game.UserProfileSavePath = "AppData\\Local\\Game\\Saves"; | Relative path from user profile (e.g. C:\Users\ZeroFox) to game's save path | Used to provide some extra functionality (open/delete/copy over to Nucleus Environment). In context = Context.UserProfileSavePath 
+Game.DocumentsConfigPath = "Path\\Here"; | Relative path from user document folder (e.g. C:\Users\ZeroFox\Documents) to game's config path | Used to provide some extra functionality (open/delete/copy over to Nucleus Environment). Use this when the game uses Documents to store game files. In context = Context.DocumentsConfigPath 
+Game.DocumentsSavePath = "Path\\Here"; | Relative path from user document folder (e.g. C:\Users\ZeroFox\Documents) to game's save path. \| Used to provide some extra functionality (open/delete/copy over to Nucleus Environment). Use this when the game uses Documents to store game files. In context = Context.DocumentsSavePath
+
+
+### Handlers
+
+***This is needed:***
+Line | Description
+---- | ----------
+Game.HandlerInterval = 100; | The interval in milliseconds the Handler should be updated at. Set it to 0 to disable updating (will lose all functionality that depends on ticks).
+
+***Higher time could help low spec PCs:***
```

Comment:
I like the title change here, so go ahead and keep it. Still needs to change to H4 heading though :) `#### Higher time could help low spec PCs`